features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/textcomposer/MessageComposerPresenter.kt

```diff
@@ -48,6 +51,11 @@ class MessageComposerPresenter @Inject constructor(
             mutableStateOf(MessageComposerMode.Normal(""))
         }
 
+        val dmMember by room.getDmMember().collectAsState(initial = null)
+
+        // Hide the composer if it's a DM and the other user is ignored
+        val isVisible = (dmMember?.isIgnored?.not()).orTrue()
```

Comment:
Why? Is it a requirement from product?
Otherwise I'd keep it for now, keep it simple.

Reply:
It's mentioned in the issue, but after talking to the iOS team, it seems like we're postponing implementing this until we also take power levels into account. I'll revert it.